components/stacks-network/src/chains_coordinator.rs

```diff
@@ -380,10 +390,10 @@ pub async fn start_chains_coordinator(
                     }
                 }
 
-                let known_tip = match &chain_event {
+                let stacks_block_update = match &chain_event {
```

Comment:
Renamed this variable to a more meaningful name.
It's used a lot in the next lines